src/GraphbackClient.ts

```diff
@@ -1,21 +1,21 @@
-import { GraphQLBackendCreator } from "graphback";
-import { sourceModule } from "./utils";
-import { transpile } from "typescript";
```

Comment:
Sure

Reply:
Actually is not about removing typescript but moving it from dependencies to devDependencies

Reply:
@b1zzu I tried to remove but it is used by the tests for some reason

Reply:
I get it

Reply:
If it's used by some tests, it's fine, we should move it to devDependencies not completely remove it from the `packages.json` because otherwise, we can't even build the testx library, but the users that will download the compiled testx library will not have to download also typescript, that's the idea